Source/AwsCommonRuntimeKit/event-stream/EventStreamMessage.swift

```diff
@@ -111,16 +112,16 @@ extension EventStreamMessage {
                         aws_event_stream_add_string_header(
                             rawHeaders,
                             headerName,
-                            UInt8(header.name.count),
+                            headerNameLength,
                             $0,
-                            UInt16(value.count),
+                            UInt16(value.utf8.count),
```

Comment:
above this diff, `if value.count > EventStreamHeader.maxValueLength` needs to also use utf8.count